src/dodal/beamlines/i19_2.py

```diff
@@ -116,3 +117,11 @@ def panda() -> HDFPanda:
         prefix=f"{PREFIX.beamline_prefix}-EA-PANDA-01:",
         path_provider=get_path_provider(),
     )
+
+
+@device_factory()
+def fastcs_eiger() -> EigerDetector:
+    return EigerDetector(
+        prefix=PREFIX.beamline_prefix,
```

Comment:
Should: it would be better to have the suffixes explicitely set here 
`-EA-EIGER-01:` for drv_suffix
`-EA-EIGER-01:OD:` for hdf_suffix